Sources/HTMLEntities/String+HTMLEntities.swift

```diff
@@ -0,0 +1,338 @@
+    "\u{03B4}":"&delta;","\u{03B5}":"&epsilon;","\u{03B6}":"&zeta;","\u{03B7}":"&eta;",
+    "\u{03B8}":"&theta;","\u{03B9}":"&iota;","\u{03BA}":"&kappa;","\u{03BB}":"&lambda;",
+    "\u{03BC}":"&mu;","\u{03BD}":"&nu;","\u{03BE}":"&xi;","\u{03BF}":"&omicron;",
+    "\u{03C0}":"&pi;","\u{03C1}":"&rho;","\u{03C2}":"&sigmaf;","\u{03C3}":"&sigma;",
+    "\u{03C4}":"&tau;","\u{03C5}":"&upsilon;","\u{03C6}":"&phi;","\u{03C7}":"&chi;",
+    "\u{03C8}":"&psi;","\u{03C9}":"&omega;","\u{03D1}":"&thetasym;","\u{03D2}":"&upsih;",
+    "\u{03D6}":"&piv;","\u{2022}":"&bull;","\u{2026}":"&hellip;","\u{2032}":"&prime;",
+    "\u{2033}":"&Prime;","\u{203E}":"&oline;","\u{2044}":"&frasl;","\u{2118}":"&weierp;",
+    "\u{2111}":"&image;","\u{211C}":"&real;","\u{2122}":"&trade;","\u{2135}":"&alefsym;",
+    "\u{2190}":"&larr;","\u{2191}":"&uarr;","\u{2192}":"&rarr;","\u{2193}":"&darr;",
+    "\u{2194}":"&harr;","\u{21B5}":"&crarr;","\u{21D0}":"&lArr;","\u{21D1}":"&uArr;",
+    "\u{21D2}":"&rArr;","\u{21D3}":"&dArr;","\u{21D4}":"&hArr;","\u{2200}":"&forall;",
+    "\u{2202}":"&part;","\u{2203}":"&exist;","\u{2205}":"&empty;","\u{2207}":"&nabla;",
+    "\u{2208}":"&isin;","\u{2209}":"&notin;","\u{220B}":"&ni;","\u{220F}":"&prod;",
+    "\u{2211}":"&sum;","\u{2212}":"&minus;","\u{2217}":"&lowast;","\u{221A}":"&radic;",
+    "\u{221D}":"&prop;","\u{221E}":"&infin;","\u{2220}":"&ang;","\u{2227}":"&and;",
+    "\u{2228}":"&or;","\u{2229}":"&cap;","\u{222A}":"&cup;","\u{222B}":"&int;",
+    "\u{2234}":"&there4;","\u{223C}":"&sim;","\u{2245}":"&cong;","\u{2248}":"&asymp;",
+    "\u{2260}":"&ne;","\u{2261}":"&equiv;","\u{2264}":"&le;","\u{2265}":"&ge;",
+    "\u{2282}":"&sub;","\u{2283}":"&sup;","\u{2284}":"&nsub;","\u{2286}":"&sube;",
+    "\u{2287}":"&supe;","\u{2295}":"&oplus;","\u{2297}":"&otimes;","\u{22A5}":"&perp;",
+    "\u{22C5}":"&sdot;","\u{2308}":"&lceil;","\u{2309}":"&rceil;","\u{230A}":"&lfloor;",
+    "\u{230B}":"&rfloor;","\u{2329}":"&lang;","\u{232A}":"&rang;","\u{25CA}":"&loz;",
+    "\u{2660}":"&spades;","\u{2663}":"&clubs;","\u{2665}":"&hearts;","\u{2666}":"&diams;",
+
+    // Markup-significant and internationalization characters
+    // https://www.w3.org/TR/html4/sgml/entities.html#h-24.4.1
+    "\u{0022}":"&quot;","\u{0026}":"&amp;","\u{003C}":"&lt;","\u{003E}":"&gt;",
+    "\u{0152}":"&OElig;","\u{0153}":"&oelig;","\u{0160}":"&Scaron;","\u{0161}":"&scaron;",
+    "\u{0178}":"&Yuml;","\u{02C6}":"&circ;","\u{02DC}":"&tilde;","\u{2002}":"&ensp;",
+    "\u{2003}":"&emsp;","\u{2009}":"&thinsp;","\u{200C}":"&zwnj;","\u{200D}":"&zwj;",
+    "\u{200E}":"&lrm;","\u{200F}":"&rlm;","\u{2013}":"&ndash;","\u{2014}":"&mdash;",
+    "\u{2018}":"&lsquo;","\u{2019}":"&rsquo;","\u{201A}":"&sbquo;","\u{201C}":"&ldquo;",
+    "\u{201D}":"&rdquo;","\u{201E}":"&bdquo;","\u{2020}":"&dagger;","\u{2021}":"&Dagger;",
+    "\u{2030}":"&permil;","\u{2039}":"&lsaquo;","\u{203A}":"&rsaquo;","\u{20AC}":"&euro;"
+]
+#endif
+
+private let html4decode: [String: Character] = [
```

Comment:
I thought about doing that, but extensions do not allow for non-declarative statements outside of the extension. Furthermore, adding of non-computed properties to the String class is not allowed; only additional computed properties are allowed.